qiskit_nature/drivers/second_quantization/gaussiand/gaussian_forces_driver.py

```diff
@@ -75,9 +75,10 @@ def __init__(
         # If running from a jcf we need Gaussian™ 16 so check if we have a
         # valid install.
         if self._logfile is None:
-            check_valid()
+            _optionals.HAS_GAUSSIAN.require_now("GaussianForcesDriver __init__")
 
     @staticmethod
+    @_optionals.HAS_GAUSSIAN.require_in_call("GaussianForcesDriver from_molecule")
```

Comment:
If you don't supply an argument in the decorator form, it should use the qualified name of the method automatically.  If you prefer, this line is equivalent to decorating with `@_optionals.HAS_GAUSSIAN.require_in_call` (i.e. without "calling" the decorator)

Reply:
Oh that is helpful. Thanks for the tip. I will change it.